client/layout/masterbar/logged-in.jsx

```diff
@@ -201,8 +241,10 @@ export default connect(
 			hasMoreThanOneSite: getCurrentUserSiteCount( state ) > 1,
 			user: getCurrentUser( state ),
 			isSupportSession: isSupportSession( state ),
-			isMigrationInProgress: !! isSiteMigrationInProgress( state, siteId ),
+			isMigrationInProgress: !! isSiteMigrationInProgress( state, currentSelectedSiteId ),
```

Comment:
I'm not sure why we're doing the `!!` here, as `isSiteMigrationInProgress` already returns a boolean – we're doing it in a couple of other places. Not a killer I guess.

Reply:
I think this is a leftover from a previous iteration on the code and looking at other places in Calypso, it's generally to ensure the return value is `bool`. 
Seems this is more of a code-style and making sure the value is in the preferred data type.

We can always change it if there's desire to unify code styles in the future.

Reply:
Cool with me!